### ISSUE TITLE
Cannot update to latest Android SDK version 2.0.0

### ISSUE THREAD

**Issue by L3K0V** [parse-community/Parse-SDK-Android#1124]
### New Issue Checklist

- [x] I am not disclosing a [vulnerability](https://github.com/parse-community/Parse-SDK-Android/security/policy).
- [x] I am not just asking a [question](https://github.com/parse-community/.github/blob/main/SUPPORT.md).
- [x] I have searched through [existing issues](https://github.com/parse-community/Parse-SDK-Android/issues?q=is%3Aissue).
- [x] I can reproduce the issue with the latest version of [Parse Server](https://github.com/parse-community/parse-server/releases) and the [Parse Android SDK](https://github.com/parse-community/Parse-SDK-Android/releases). 

### Issue Description

```
Could not find com.github.parse-community.Parse-SDK-Android:parse:2.0.0.
```

### Steps to reproduce

Update the SDK version of the Android to 2.0.0

### Actual Outcome
Cannot fetch the artefacts

### Expected Outcome
To be able to fetch the artefacts

### Environment

Parse Android SDK
- SDK version: `2.0.0`
- Operating system version: `macOS`

Server
- Parse Server version: `latest`
- Operating system: `macOS`
- Local or remote host (AWS, Azure, Google Cloud, Heroku, Digital Ocean, etc): `local`

Database
- System (MongoDB or Postgres): `MongoDB`
- Database version: `4`
- Local or remote host (MongoDB Atlas, mLab, AWS, Azure, Google Cloud, etc): `latest`

### Logs

```
Execution failed for task ':admin:mergeDebugNativeLibs'.
> Could not resolve all files for configuration ':admin:debugRuntimeClasspath'.
   > Could not find com.github.parse-community.Parse-SDK-Android:parse:2.0.0.
     Required by:
         project :admin > project :shared
   > Could not find com.github.parse-community.Parse-SDK-Android:coroutines:2.0.0.
     Required by:
         project :admin > project :shared
   > Could not find com.github.parse-community.Parse-SDK-Android:ktx:2.0.0.
     Required by:
         project :admin > project :shared
   > Could not find com.github.parse-community.Parse-SDK-Android:fcm:2.0.0.
     Required by:
         project :admin > project :shared
```

**Comment by mtrezza** [comment]
If the 2.0 SDK is currently really not downloadable, can we take what's necessary from #1122 and fix this issue separately?

**Comment by L3K0V** [comment]
@mtrezza do you want me to take care of it? The new PR will overlap a lot with the existing #1122

**Comment by mtrezza** [comment]
Yes please, that would be fantastic. I would consider not being able to download the SDK a priority issue.

**Comment by parseplatformorg** [comment]
🎉 This pull request has been released in version [2.0.1](https://github.com/parse-community/Parse-SDK-Android/releases/tag/2.0.1)

**Comment by parseplatformorg** [comment]
🎉 This pull request has been released in version [2.0.1](https://github.com/parse-community/Parse-SDK-Android/releases/tag/2.0.1)
